src/day-1-toy/func.py

```diff
@@ -1,6 +1,11 @@
 # Write a function is_even that will return true if the passed in number is even.
-
+def is_even(num):
+    if int(num) % 2 == 0: 
+        return "Even!"
+    else:
+        return "Odd"
 # Read a number from the keyboard
```

Comment:
Be careful of the output: the instructions says to return `True` if the passed in number is even, not an "Even!" string. Otherwise, this is great